### PR TITLE
Document that type-feature labels do not need to be paired with version labels

### PR DESCRIPTION
**Pull request by erlend-aasland**
----
📚 Documentation preview 📚: https://cpython-devguide--1259.org.readthedocs.build/

### CODE REVIEW

File: triage/labels.rst [hunk 1]
```diff
@@ -26,6 +26,8 @@ These labels are used to specify the type of issue:
 * :gh-label:`type-crash`: for hard crashes of the interpreter, possibly with a
   core dump.
 * :gh-label:`type-feature`: for feature requests or enhancements.
+  Feature requests do not need :ref:`Version labels`;
```
Comment (hugovk) [inline triage/labels.rst:29]: Avoid the capital V in the middle of a sentence:

```suggestion
  Feature requests do not need :ref:`version labels <Version labels>`;
```

Comment (erlend-aasland) [reply triage/labels.rst:29]: Good catch! BTW; I found the capitalised anchor names weird. Is this a new thing? I can't remember seeing it in the docs before.

Comment (hugovk) [reply triage/labels.rst]: Hmm, I'm not sure, let's ask @AA-Turner.

**Review by hugovk — APPROVED** [review]

File: triage/labels.rst (hunk 1 above)
Comment (erlend-aasland) [reply triage/labels.rst]: Well, if it is ok for Adam, I'd like to add the anchor in lower-case 🙂

**Comment by erlend-aasland** [comment]
Thanks for the reviews!

**Pull request merged**